src/cli/new-watcher.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const path = require('path');
+
+const watcherPlaceholderPath = require.resolve('../watcher-placeholder.txt');
```

Comment:
Remember to use `path.join` for cross-platform compatibility.

Reply:
I think `require` manages that by default